tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V4Isolated/Program.cs

```diff
@@ -1,7 +1,13 @@
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
+        .ConfigureLogging(logging =>
+        {
+            logging.AddConsole();
+            logging.SetMinimumLevel(LogLevel.Trace);
+        })
```

Comment:
You have this in the "disabled" case, do you want them to be the same?
```suggestion
            logging.AddFilter("*", LogLevel.Trace);
        })
```